src/main.rs

```diff
@@ -0,0 +1,58 @@
+use clap::{arg, value_parser, Command};
+
+fn cli() -> Command {
+    Command::new("state-reconstruct")
+        .about("zkSync state reconstruction tool")
+        .subcommand_required(true)
+        .arg_required_else_help(false)
+        .subcommand(
+            Command::new("import")
+                .about("Import state")
```

Comment:
```suggestion
            Command::new("reconstruct")
                .about("reconstruct state")
```
Nit: maybe "reconstruct" since we are doing a lot more than just importing the L1 state.

Reply:
Initially my thinking was that there would be `export` as well and in that case, I don't think `deconstruct` is necessarily that sound expression for the behaviour :smile: 

...but, revisiting requirements, I think officially there won't be export functionality so I'm ok with this. I'm thinking of hidden `export` command though, because we do need it for development purposes.